tests/e2e/common-operator-integ-suite.sh

```diff
@@ -234,8 +234,12 @@ if [ "${SKIP_BUILD}" == "false" ]; then
     # Install OLM in the cluster because it's not available by default in kind.
     ${OPERATOR_SDK} olm install
 
-    # Wait to operatorhub pod to be running
-    ${COMMAND} wait --for=condition=ready pod -l olm.catalogSource=operatorhubio-catalog -n olm --timeout=5m
+    # List the condition of the catalogsource to check if it's ready
+
+
+    # Wait for for the CatalogSource to be CatalogSource.status.connectionState.lastObservedState == READY
+    timeout --foreground -v -s SIGHUP -k ${TIMEOUT} ${TIMEOUT} bash --verbose -c \
+      "until ${COMMAND} get CatalogSource operatorhubio-catalog -n olm -o jsonpath='{.status.connectionState.lastObservedState}' | grep -q READY; do sleep 5; done && echo 'The CatalogSource is Ready.'"
```

Comment:
This can be replaced with something like: 
```
k wait catalogsource operatorhubio-catalog --for 'jsonpath={.status.connectionState.lastObservedState}=READY'
```

Reply:
Done